client/staging-site/components/staging-site/index.tsx

```diff
@@ -10,15 +10,12 @@ import './style.scss';
 
 const StagingSite = () => {
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) ) ?? 0;
-	const hasAtomicFeature = useSelector( ( state ) =>
-		siteHasFeature( state, siteId, WPCOM_FEATURES_ATOMIC )
-	);
 	const hasStagingSitesFeature = useSelector( ( state ) =>
 		siteHasFeature( state, siteId, FEATURE_SITE_STAGING_SITES )
 	);
 	const isWpcomStagingSite = useSelector( ( state ) => isSiteWpcomStaging( state, siteId ) );
 
-	if ( ! hasAtomicFeature && ! hasStagingSitesFeature ) {
+	if ( ! hasStagingSitesFeature ) {
```

Comment:
@arthur791004 I removed the `! hasAtomicFeature` since I thought `! hasStagingSitesFeature` covers. Let me know if I'm missing anything.

Reply:
Yep. I think it's enough as well 🙂